store/sqlstore/store.go

```diff
@@ -909,7 +909,23 @@ func (ss *SqlStore) AlterColumnTypeIfExists(tableName string, columnName string,
 	return true
 }
 
-func (ss *SqlStore) AlterColumnDefaultIfExists(tableName string, columnName string, mySqlColDefault *string, postgresColDefault *string) bool {
+func (ss *SqlStore) RemoveDefaultIfColumnExists(tableName, columnName string) bool {
+	if !ss.DoesColumnExist(tableName, columnName) {
+		return false
+	}
+
+	_, err := ss.GetMaster().ExecNoTimeout("ALTER TABLE " + tableName + " ALTER COLUMN " + columnName + " DROP DEFAULT")
+	if err != nil {
+		mlog.Critical("Failed to drop column default", mlog.String("table", tableName), mlog.String("column", columnName), mlog.Err(err))
+		time.Sleep(time.Second)
```

Comment:
I've always wondered why do we sleep for a second here? :P

Reply:
Likewise :) And what's worse I keep copying it for some inner sense of consistency but fully knowing it's most probably totally unnecessary :/

Reply:
I wouldn't be surprised if it's to allow the log file to be flushed or something like that. The server didn't used to shut down as gracefully before as it does now.